README.md

```diff
@@ -1,242 +1,278 @@
-# Quint Lang
+<div align="center">
+
+<!-- Title -->
+<img src="./logos/quint-logo.png" alt="Quint Lang Logo" width="120"/>
+<h1>Quint Lang</h1>
+
+<!-- Menu -->
+<p>
+    <a href="#installation">Installation</a> •
+    <a href="./doc#readme">Documentation</a> •
+    <a href="#community">Community</a>
+</p>
+
+<!-- Badges -->
+<p>
+    <a href="https://github.com/informalsystems/quint/actions">
+        <img
+            src="https://github.com/informalsystems/quint/actions/workflows/main.yml/badge.svg"
+            alt="build badge">
+    </a>
+    <a href="https://marketplace.visualstudio.com/items?itemName=informal.quint-vscode">
+        <img
+            src="https://img.shields.io/visual-studio-marketplace/v/informal.quint-vscode?color=10b0f2&label=VSCode"
+            alt="Visual Studio Marketplace Version">
+    </a>
+    <a href="https://www.npmjs.com/package/@informalsystems/quint">
+        <img
+            src="https://img.shields.io/npm/v/@informalsystems/quint"
+            alt="npm (scoped)">
+    </a>
+</p>
+</div>
+
+
+## Overview
+
+Quint is a modern specification language that is a particularly good fit for
+distributed systems, such as blockchain protocols, distributed databases, and
+p2p protocols. Quint combines the robust theoretical basis of the [Temporal
+Logic of Actions (TLA)][TLA] with state-of-the-art static analysis and
+development tooling.
+
+### Example code in Quint :mrs_claus: :gift: :santa:
+
+``` bluespec
+module secret_santa {
+  import basicSpells.* from "../../spells/basicSpells"
+  import commonSpells.* from "../../spells/commonSpells"
+
+  const participants: Set[str]
+
+  /// get(recipient_for_santa, S) is the recipient for secret santa S
+  var recipient_for_santa: str -> str
+
+  /// the bowl of participants, containig a paper piece for each participant name
+  var bowl: Set[str]
+
+  val santas = recipient_for_santa.keys()
+  val recipients = recipient_for_santa.mapValuesSet()
```

Comment:
I don't feel strongly here but I don't agree that inlining definitions when we have a clear operator for it is preferable, esp. not when the point is show a very simple spec. 

That said, the name of the operator is not very nice, and if it were `values` I would feel stronger about it :)

I'll defer to @bugarela here, who introduced use of `mapValuesSet`.

Reply:
I agree with @shonfeder on this, in the sense that I'd keep this as is if the name of the operator was better (like `values()`). However, in this case, I'd go with Igor's suggestion. The reason why I introduced `mapValuesSet` was because I was repeating the `map` thing in two different places, but with the use of auxiliary definitions, I think the `map` expression is clearer than this odd-named helper.

Reply:
Sounds good!